src/cli/commands/files/cat.js

```diff
@@ -36,7 +36,7 @@ module.exports = Command.extend({
           throw (err)
         }
         res.on('data', (data) => {
-          data.stream.pipe(process.stdout)
+          data.content.pipe(process.stdout)
         })
```

Comment:
This will be changed to make res the stream instead, once we have the interface-ipfs-core for cat merged too